proto/intertx/tx.proto

```diff
@@ -24,17 +26,29 @@ message MsgRegisterAccount {
   // the channel by which the packet will be sent
   string source_channel = 3 [(gogoproto.moretags) = "yaml:\"source_channel\""];
 
-  // Timeout height relative to the current block height.
-  // The timeout is disabled when set to 0.
-  ibc.core.client.v1.Height timeout_height = 4
-      [(gogoproto.moretags) = "yaml:\"timeout_height\"", (gogoproto.nullable) = false];
-
-  // Timeout timestamp (in nanoseconds) relative to the current block timestamp.
-  // The timeout is disabled when set to 0.
-  uint64 timeout_timestamp = 5 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
   string owner=6;
 }
 
 message MsgRegisterAccountResponse {}
 
+// MsgSend represents a message to send coins from one account to another.
```

Comment:
From one account to another account on the same chain? A different chain?

Reply:
Updated :+1: